index.html

```diff
@@ -6108,11 +6135,126 @@ <h3>Algorithm</h3>
+            <td><code>String</code></td>
+            <td><code>str</code></td>
+          </tr>
+          <tr>
+            <th><a>JsonLdRecord</a></th>
+            <th><code><a data-cite="INFRA#ordered-map">Map</a></code></th>
+            <td><code>Object</code></td>
+            <td><code>dict</code></td>
+          </tr>
+          <tr>
+            <th><a data-cite="WebIDL##idl-sequence">sequence</a></th>
+            <th><code><a data-cite="INFRA#list">list</a></code></th>
+            <td><code>Array</code></td>
+            <td><code>list</code></td>
+          </tr>
+          <tr>
+            <th><code><a data-cite="WebIDL##idl-long">long</a></code></th>
+            <th><code><a data-cite="XMLSCHEMA11-2#integer">integer</a></code> [[?XMLSCHEMA11-2]]</th>
+            <td><code>Number</code></td>
+            <td><code>int</code></td>
+          </tr>
+          <tr>
+            <th><code><a data-cite="WebIDL##idl-double">double</a></code></th>
+            <th><code><a data-cite="XMLSCHEMA11-2#double">double</a></code> [[?XMLSCHEMA11-2]]</th>
+            <td><code>Number</code></td>
+            <td><code>float</code></td>
+          </tr>
+          <tr>
+            <th><code><a data-cite="WebIDL##idl-boolean">boolean</a></code></th>
+            <th><code><a data-cite="INFRA#boolean">boolean</a></code></th>
+            <td><code>Boolean</code></td>
+            <td><code>bool</code></td>
+          </tr>
+          <tr>
+            <th><code><strong>null</strong></code></th>
+            <th><code><a data-cite="INFRA#nulls">null</a></code></th>
+            <td><code>Null</code></td>
+            <td><code>None</code></td>
+          </tr>
+        </tbody>
+      </table>
+
+      <p>
+        The mappings used for a particular implementation are out of scope for
+        this specification.
+      </p>
+    </section>
+
+    <section id="serialization-and-deserialization">
+      <h4>Serialization and Deserialization</h4>
+    <ins cite="#change_5">
+
+      <p>This section defines the requirements for serializing and deserializing
+        JSON to and from the <a>internal representation</a>, as appropriate,
+        based on the provided or requested <a>Content-Type</a>.
+        A <a>Content-Type</a> of <code>application/json</code>,
+        or media type with a <code>+json</code> suffix as defined in [[RFC6839]]
+        MUST serialize or deserialize to a format consistent with the JSON Grammar
+        as specified in [[RFC8259]].</p>
+
+      <p class="note">This specification does not define serializations of the <a>internal representation</a>
+        for other media types, which may be defined by other specifications.</p>
+
+      <p class="note">See <a href="#html-content-algorithms" class="sectionRef"></a>
+        for details on extracting JSON from HTML.</p>
+
+      <dl>
+        <dt>To transform a string conforming to JSON [[RFC8259]],</dt>
```

Comment:
```suggestion
        <dt>To transform a JSON-conforming string [[RFC8259]],</dt>
```
avoids the confusing use of "to" after "transform" (reads like "JSON" is the *output* of the transformation)